src/org/openboxprotocol/protocol/OpenBoxHeaderMatch.java

```diff
@@ -0,0 +1,113 @@
+package org.openboxprotocol.protocol;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openboxprotocol.types.Masked;
+import org.openboxprotocol.types.ValueType;
+
+public class OpenBoxHeaderMatch implements HeaderMatch {
+
+	private Map<HeaderField<?>, ValueType<?>> fields;
+	private Map<HeaderField<?>, ValueType<?>> masks;
```

Comment:
This means we should only have the fields member? 
Is it enough to have setFields() and setWildcard() in the Builder interface?